README.md

```diff
@@ -1,32 +1,167 @@
+- [Paul Butler](https://paulbutler.org), a self-proclaimed quant out of NYC whose website 
+https://ranked.vote helped us model our synthetic data for this project
+- [Aaron Hamlin](https://www.linkedin.com/in/aaronhamlin/), the Executive Director of [The Center for Election Science](https://electionscience.org) who 
+put us in touch with Paul & generally helped us navigate the complicated world of voting systems.
+  
+## What is Ranked Choice Voting?
+Ranked Choice Voting is ["an electoral system in which voters rank candidates by preference on their ballots."](https://ballotpedia.org/Ranked-choice_voting_(RCV)) A typical ballot in an RCV election looks like this: 
+
+<p align="center">
+<img src="./img/rcv_ballot_example.png" width="420" height="380"/>
+</p>
+
+As you can see, instead of voting for a single candidate, voters are asked to grade all possible candidates 
+according to preference.
+
+**Note:** sites like [Election Science](https://electionscience.org/library/the-spoiler-effect/) also call RCV voting 
+"Instant Runoff Voting."
+
+## What is the Spoiler Effect?
+["In election parlance, a spoiler is a non-winning candidate whose presence on the ballot affects which candidate 
+wins. In mathematical terms, the spoiler effect is when a voting method exhibits failure of a property known as 
+independence of irrelevant alternatives."](https://electionscience.org/library/the-spoiler-effect/)
+
+Check out [this video](https://www.youtube.com/watch?v=JtKAScORevQ) for a more in depth explanation. 
+
+Below is a nice example to read through: 
+
+<p align="center">
+<img src="./img/irv_walkthrough.png" width="420" height="380"/>
+</p>
+
+Even though the Spoiler Effect can happen in almost any election system, [proponents of Ranked Choice Voting say](http://archive3.fairvote.org/reforms/instant-runoff-voting/irv-and-the-status-quo/spoiler-effect/) that it is 
+so uncommon a phenomenon that voters shouldn't have to worry about it. However, a good number of people [say the 
+opposite.](https://psephomancy.medium.com/ranked-choice-voting-doesnt-fix-the-spoiler-effect-80ed58bff72b) So, 
+what's really going on? *That's the question this project aims to answer...*
+
+tktk -- diff btwn exhausted ballot & spoiled effect
 
 ## Methods
-### Obstacles & how we dealt with them:
-- Exhausted ballots: https://democracyjournal.org/arguments/ranked-choice-voting-is-not-the-solution/
-- Noise
-- Probabilities of vote spread across possible candidates
+### Data & Features
+Finding data with which to train our model with was *difficult*. There are hardly any public datasets that include 
+1+ spoiled elections (in an RCV system), let alone tens of thousands. For this reason, we chose one of the most famous 
+spoiled, RCV elections off of which we could model synthetic data: [Burlington's 2009 mayoral election.](http://bolson.org/~bolson/2009/20090303_burlington_vt_mayor.html) In this election, a candidate named Kurt Wright acted as a Spoiler. 
```

Comment:
would this read clearer? :

For this reason, we chose one of the most famous 
spoiled, RCV elections as a model for our synthetic data:

Reply:
Oo yes, I like it, thanks!